tests/std/tests/P1518R2_stop_overconstraining_allocators/test.compile.pass.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <deque>
+#include <forward_list>
+#include <list>
+#include <map>
+#include <memory_resource>
+#include <queue>
+#include <set>
+#include <stack>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+using namespace std;
+
+extern pmr::monotonic_buffer_resource mr;
+
+template <template <class...> class Ctr>
+struct SfinaeTester {
+    template <class... Args>
+    static auto f(int, Args&&... args) -> decltype(Ctr(static_cast<Args&&>(args)...)) {
+        return Ctr(static_cast<Args&&>(args)...);
+    }
+    template <class... Args>
+    static void f(long, Args&&... /*args*/) {}

```

Comment:
Conventionally those are simply omitted
```suggestion
    static void f(long, Args&&...) {}
```